omnibus/config/software/datadog-dogstatsd.rb

```diff
@@ -31,6 +31,12 @@
   end
 
   # move around bin and config files
+  if windows?
+    mkdir "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
+    copy 'bin/dogstatsd/dogstatsd.exe', "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/DataDog/datadog-agent/bin/agent"
+  else
+    copy 'bin/dogstatsd/dogstatsd', "#{install_dir}/bin"
+  end
   copy 'bin/dogstatsd/dogstatsd', "#{install_dir}/bin"
```

Comment:
I guess this line is not necessary anymore, since it's already done in the prior `if` statement